utils/rsync-nbsd.sh

```
@@ -0,0 +1,9 @@
+TDNS_SRC=/Users/johani/src/git/tdns
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)